src/libraries/shims/manual/System.Data.csproj

```diff
@@ -2,4 +2,8 @@
   <ItemGroup>
     <Compile Include="System.Data.forwards.cs" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <!-- System.Data.SqlClient is not live built in anymore -->
+    <ReferencePath Include="$(ArtifactsBinDir)ref\$(NetCoreAppCurrent)\System.Data.SqlClient.dll" />
```

Comment:
Should we copy to runtime as well?   That would avoid this workaround as well as other PackageReferences.

Reply:
The runtime/testhost directory currently contains only shared framework assets. I'm a bit worried that adding System.Data.SqlClient there conflicts with the principle of testing the live built shared framework. I admit, we are also putting OOB assemblies there but that's just for convenience atm.

Reply:
> we are also putting OOB assemblies there but that's just for convenience atm.

Right, the runtime/testhost contains a pile of stuff that should not be there. So this is a question of whether to be consistent with the convenience we have there today, or whether to do work towards cleaning it up. I went back and forth on this. Neither of these options are 100% pretty.

Reply:
> or whether to do work towards cleaning it up

I started cleaning it up when I refactored the testing logic in corefx and removed all testing assets. Currently there shouldn't be any cruft besides the OOB assets, CoreRun which was brought back for profiling/performance testing.

I planned to remove the OOB assets from there as well when we did the work to make our test libraries consumable by Mono. This isn't a high prio anymore but I still tend towards making the OOB libraries app local in the test output folders instead of having them in the shared framework testhost.

Reply:
> I planned to remove the OOB assets

I agree that this is cleaner and it is what the current shape of the PR is trying to do: I have added System.Data.SqlClient package references for the two tests that need them. But that it is colliding with other other tricks that the build plays and causing Mono and allConfigurations legs to fail.

Reply:
> Should we copy to runtime as well?

Trying this alternative now.

Reply:
OK. Do you understand the Mono failures? https://dnceng.visualstudio.com/public/_build/results?buildId=506541&view=ms.vss-test-web.build-test-results-tab&runId=16074090&resultId=110670&paneView=debug

Reply:
I believe that some files are getting overwritten during the build with a wrong copy.

Reply:
> Should we copy to runtime as well?

I wasn't following, why do we need to copy this into the runtime/testhost folder?